src/WCSimPMTObject.cc

```diff
@@ -1965,3 +2011,46 @@ G4double PMT3inch::GetDarkRateConversionFactor(){
   const G4double factor = 1.367;
   return factor;
 }
+
+G4int PMT3inch::GetNbOfQEDefined(){
+  const G4int factor = 20;
+  return factor;
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////
+// COMBINED PMTS CLASS
+// Goal is to instantiate purely virtual WCSimPMTObject Classes to contained an extended
+//  QE information of several PMTs collection
+// Useful for applying QE in stacking action when several PMTs collection are defined
```

Comment:
@federiconova do you have any comments on this?

Reply:
Hi, sorry, I am happy to comment but what are the incriminated lines? I only see four commented lines.

Reply:
I see that in `WCSimDetectorConfigs` we make use of the combined PMT QE for the OD configuration. Does this affect the QE of the ID PMTs at all?

Reply:
Mahdi I think you had implemented these lines but they are actually never used. In DetectorConfigs, isCombinedPMTCollectionDefined=true. So then in StackingAction the code uses GetPMTQE instead of GetStackingPMTQE. So the WCSimBasicPMT object is then never used (in WCSimPMTQE). I feel it should be safe to just remove the BasicPMT class (to which these comments refer) and all references to it.